src/main/webapp/project/NewProjectDocuments.js

```diff
@@ -104,15 +104,17 @@ export const NewProjectDocuments = hh(class NewProjectDocuments extends Componen
     }
 
     let documents = this.props.files;
-
     let errors = false;
-
+    
     return (
 
       WizardStep({
         title: this.props.title, step: this.props.step, currentStep: this.props.currentStep,
-        errorMessage: !this.props.errors.attestation && this.props.generalError ? 'Please check previous steps' : 'Please complete all required fields',
-        error: this.props.generalError || this.props.errors.attestation
+        errorMessage: !this.props.errors.attestation && this.props.generalError ? 
+        'Please check previous steps' : !this.props.submitError ? 
+        'Please complete all required fields' : 
+        'Something went wrong. Please try again.',
+        error: this.props.generalError || this.props.errors.attestation || this.props.submitError
```

Comment:
Nit: the logic here, along with everything aligned at the same level, is very hard to follow. I suggest breaking this down into some intermediate variables that might help clarify what the conditions are.